applications/RANSApplication/custom_processes/rans_check_scalar_bounds_process.h

```diff
@@ -0,0 +1,122 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_RANS_CHECK_SCALAR_BOUNDS_PROCESS_H_INCLUDED)
+#define KRATOS_RANS_CHECK_SCALAR_BOUNDS_PROCESS_H_INCLUDED
+
+// System includes
+#include <string>
+
+// External includes
+
+// Project includes
+#include "containers/model.h"
+#include "processes/process.h"
+
+namespace Kratos
+{
+///@addtogroup RANSApplication
+///@{
+
+///@name Kratos Classes
+///@{
+
+/**
+ * @brief Checks lower and upper bounds of a scalar
+ *
+ * This process checks lower and upper bounds of a variable in nodes in a given modelpart
+ *
+ */
+
+class KRATOS_API(RANS_APPLICATION) RansCheckScalarBoundsProcess : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    using NodeType = ModelPart::NodeType;
```

Comment:
I think you are not using this typename.

Reply:
removed process